### PR TITLE
Mail chimp

### PR DESCRIPTION
**Pull request by Aleksei-Zaichenko**
Loom video: https://www.loom.com/share/de2b093a3be140bfaac715badc88b8bd

Modified Step1.js in MasterForm.js. I added validation for each input field, introduced a new field (country code) and if a new user chooses "yes" for sign up for newsletter field, this form makes a post request to back-end api and then to MailChimp api. 

Created a new rest api endpoint to accept a new user`s credentials and then send them to MailChimp api.

### CODE REVIEW

**Comment by brighamband** [comment]
Looks great! Good work!
I'd approve it, but it says merging is blocked

File: api/urls.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 urlpatterns = [
     path("signup/", first_time_signup),
+    path("signup/subscriptionForNewsletters/",sign_up_for_newsletters ),
```
Comment (edringger) [inline api/urls.py:6]: you can probably just make this its own url path outright (ie /subscribe) but I guess it doesn't really meatter

File: src/components/registration/Step1.js [hunk 2]
```diff
@@ -52,9 +155,25 @@ function Step1(props) {
         name="phone"
         type="tel"
         placeholder="Enter phone number (digits only) ie '1234567890'"
-        value={props.state.phone}
-        onChange={props.handleChange}
+        value={formState.phone}
+        onChange={(event) => inputChange(event)}
       />
+
```
Comment (edringger) [inline src/components/registration/Step1.js:161]: I think this should be combined with the phone field. It's not very intuitive to have to put your country code in a separate box, and most people know to put their country code in with their phone number

**Review by edringger — CHANGES_REQUESTED** [review]
Looks great! The only thing I would change would be to make country code somehow be on the same line as phone number. Right now it's not very intuitive. Most people are comfortable typing in their country code as part of their phone number anyways, I think. I would just remove that part of the form for now, and add it as a separate issue to work on later, since the mail chimp part works. 

Using something like this would be cool: npmjs.com/package/react-phone-input-2

**Review by edringger — APPROVED** [review]

**Pull request merged**